utils/slide-switch/Makefile

```diff
@@ -8,15 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=slide-switch
-PKG_VERSION:=0.9.4
+PKG_VERSION:=0.9.5
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/jefferyto/openwrt-slide-switch.git
```

Comment:
.git at the end is unnecessary.

Reply:
This is the official URL GitHub gives to clone using https; I prefer using this URL.

The fact that the `.git` at the end is not required is an unofficial convenience that can be revoked at any time.

Reply:
Fair enough.